Web/Edubase.Web.UI/Assets/Scripts/GiasEdit/locationAutoSuggests.js

```diff
@@ -20,12 +20,6 @@ const locationAutoSuggest = (function() {
       hiddenFieldId: 'ParliamentaryConstituencyIdHidden',
       maxItems: 10
     },
-    {
-      textFieldId: 'CASWardName',
-      suggestions: window.caswards,
-      hiddenFieldId: 'CASWardIdHidden',
-      maxItems: 30
-    },
```

Comment:
@jonbanner - looks like i already removed it... was there somewhere else you mean with the JS?

Reply:
ah sorry missed that when looking it over.

Reply:
easy done. I tried search the assets for "cas" references, so i'm hoping i caught everything